users/migrations/0001_squashed_0005_auto_20210415_1604.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 3.1.7 on 2021-04-15 18:58
+
+import django.contrib.auth.models
+import django.contrib.auth.validators
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('users', '0001_initial'), ('users', '0002_auto_20210414_1321'), ('users', '0003_customuser_activate_date'), ('users', '0004_auto_20210415_1559'), ('users', '0005_auto_20210415_1604')]
```

Comment:
Do you really need this squashed migration? Can we get rid of it?

Reply:
Yes, sure. Is it a good or bad idea to compose a few small migrations in one squashed migration? Or it's better to save all migration history?